src/client/components/atoms/burgerButton/styles.scss

```diff
@@ -0,0 +1,55 @@
+.burger-button {
```

Comment:
The easiest way of applying BEM patter is to make the "Block" to be just 1 word. otherwise the rest of classNames will get longer and hard to read/understand in future.

suggestion: I'd just call it `Burger` (ideally `Menu`  but is already in use with other stuff)